string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,16 @@
+/* exported numVowels */
+function numVowels(string) {
+  var vCount = 0;
+  for (var i = 0; i < string.length; i++) {
+    var capLetter = string[i].toLowerCase();
```

Comment:
`capLetter` that hold a lowercase letter? Unless this letter is secretly Captain America, this variable seems to be misnamed.